packages/jaeger-ui/src/components/SearchTracePage/FileLoader.tsx

```diff
@@ -24,7 +24,11 @@ type FileLoaderProps = {
 
 export default function FileLoader(props: FileLoaderProps) {
   return (
-    <Dragger accept=".json" customRequest={props.loadJsonTraces} multiple>
+    <Dragger
+      accept=".json"
+      beforeUpload={(file, fileList) => fileList.forEach(file => props.loadJsonTraces({ file }))}
```

Comment:
maybe move this to a function and add unit test just for it?

Reply:
Sure. Would try adding a test for it, if possible

Reply:
Done!